network-api/networkapi/wagtailpages/pagemodels/profiles.py

```diff
@@ -41,3 +45,7 @@ class Profile(TranslatableMixin, models.Model):
 
     def __str__(self):
         return self.name
+
+    def save(self, *args, **kwargs):
+        self.slug = slugify(f'{self.name}-{str(self.id)}')
+        super(Profile, self).save(*args, **kwargs)
```

Comment:
**Issue** I believe the missing blank line at the end is failing CI.

Reply:
addressed in f331eca11ff6765f9e2e574c803de1f8e96a8a9b